pcdsdevices/ipm.py

```diff
@@ -49,15 +96,276 @@ def removed(self):
         """Returns ``True`` if target is removed. Diode never blocks."""
         return self.target.removed
 
-    def remove(self, moved_cb=None, timeout=None, wait=False):
-        """Moves the target out of the beam. Diode never blocks."""
-        return self.target.remove(moved_cb=moved_cb,
-                                  timeout=timeout,
-                                  wait=wait)
```

Comment:
I would argue that the `remove` method should not be removed so that we can still unambiguously move these out via lightpath or similar apps with a simple consistent interface (`device.remove() for device in devices`). The previous implementation worked under the assumption that the `IPMDiode` was never blocking and did not need to be removed, I do not know if that is a good assumption.

Reply:
The diode does not block if it's "IN" or "OUT" but it can block if it e.g. got stuck in an intermediate state.

Reply:
So should I put this method back in and have it remove both target and diode?